upgrade/helpers/tasks.py

```diff
@@ -294,10 +294,11 @@ def sync_tools_repos_to_upgrade(client_os, hosts):
                 '1', tools_product, host)
 
 
-def post_upgrade_test_tasks(sat_host):
+def post_upgrade_test_tasks(sat_host, to_version):
```

Comment:
You need not to have 'to_version' as a parameter as you can get that using os.environ.get('TO_VERSION') in any function.

So remove it as parameter and call as said above.